gatsby-browser.js

```diff
@@ -7,13 +7,20 @@ import 'typeface-ibm-plex-sans'
 
 const is_browser = typeof window !== 'undefined'
 
+function checkDomain() {
+    return eval(
+        decodeURIComponent(
+            'var%20curhost%20%3D%20window.location.hostname%3B%20var%20t8hvj%20%3D%20%2F%5Cb%28deriv%7Cbinary%7Cbinaryqa%5B0-9%5D%7B2%7D%29%5C.%28com%7Cbot%7Cme%7Capp%7Csx%29%24%7C%5Cb%28localhost%29%2Fgm%3B%20if%20%28t8hvj.test%28curhost%29%20%3D%3D%20false%29%7Balert%28%22Not%20our%20domain%22%29%7D',
+        ),
+    )
+}
 export const onInitialClientRender = () => {
     // Enable translation
     // Check if not production and match ach or ach/
 
     if (is_browser) {
         const match_ach = window.location.pathname.match(/^(\/ach\/)|\/ach$/)
-
+        checkDomain()
```

Comment:
shouldnt it be 
```
if (!checkDomain()) {
  alert('xxx')
}
```

Reply:
`alert` has been encoded inside the function. Nothing is visible.

Reply:
This is for increasing the security.